mlpf/model/training.py

```diff
@@ -912,16 +912,34 @@ def run(rank, world_size, config, args, outdir, logfile):
         checkpoint = torch.load(config["load"], map_location=torch.device(rank))
         start_epoch = checkpoint["extra_state"]["epoch"] + 1
 
+        missing_keys, strict = [], True
         for k in model.state_dict().keys():
             shp0 = model.state_dict()[k].shape
-            shp1 = checkpoint["model_state_dict"][k].shape
+            try:
+                shp1 = checkpoint["model_state_dict"][k].shape
+            except KeyError as e:
+                missing_keys.append(k)
+                continue
             if shp0 != shp1:
                 raise Exception("shape mismatch in {}, {}!={}".format(k, shp0, shp1))
 
+        if len(missing_keys) > 0:
+            _logger.warning(f"The following keys are missing in the checkpoint file {missing_keys}", color="red")
+            while True:
+                prompt = input("Do you want to try loading the model checkpoint with relaxed constraints? [y/n]")
```

Comment:
I think waiting for keyboard input might give some issues in batch jobs. How about adding a "relaxed_load" boolean option to the config?

Reply:
Good point! I totally forgot that. I can push a new commit later today.

Reply:
I have removed the user prompt. Please check.